spec/models/period_filter_spec.rb

```diff
@@ -92,4 +92,24 @@
       end
     end
   end
+
+  describe '#legal_periods' do
+    subject(:legal_periods) do
+      described_class.new.legal_periods
+    end
+
+    context 'when NO percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods.first.employee_percent).to eq(1)
+        expect(legal_periods.first.employer_percent).to eq(1)
+      end
+    end
+
+    context 'when percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods[1].employee_percent).to eq(3)
```

Comment:
Sorry I miss the comment. 

@Tr4pSt3R can you elaborate more your solution, please? I didn't understand what you're want. Just remember the `.first` and `[1]` are different methods with different outcomes:

```ruby
 2.1.2 :001 >  my_precious_array = [0, 1, 2, 3]
 => [0, 1, 2, 3]
2.1.2 :002 > my_precious_array.first
 => 0
2.1.2 :003 > my_precious_array[1]
 => 1
```
What am I missing?